frontend/src/components/board/items/RetroItem.tsx

```diff
@@ -96,6 +102,19 @@ function RetroItem(props: RetroItemProps) {
   const { userState, downvoteCard } = useContext(UserContext);
   const { currentTheme } = useContext(ColorThemeContext);
   const classes = useStyles(currentTheme);
+  const contentWithLinks = createContentWithLinks(content); 
+
+  function createContentWithLinks(content: string) {
+    // Regex for matching every kind of URLs
+    const urls = /(?:\w+:\/\/[\w.]+|[\w.]+\.\w+).*/.exec(content);
+    
+    urls?.forEach((val: string) => {
+      const url = val.indexOf('//') == -1 ? 'https://' + val : val;
```

Comment:
Does this also handle regular http links? Furthermore, `val` could have a more expressive name like `url` for example.

Reply:
It will add the https extension when there is no extension given. When the address is pointed to an address without an https:// Endpoint, the browser will point out that there is an Certification Error. 
Woukd you find it more fitting when it will allways point to http with the risk that it wont point to https ?

Reply:
No, I like your solution. Just wanted a bit more clarification. LGTM 👍